src/cache.rs

```diff
@@ -90,11 +96,37 @@ pub fn evict(max_age: usize) {
     accelerate::evict();
 }
 
+/// Evict the thread local cache.
+///
+/// This removes all memoized results from the cache whose age is larger than or
+/// equal to `max_age`. The age of a result grows by one during each eviction
+/// and is reset to zero when the result produces a cache hit. Set `max_age` to
+/// zero to completely clear the cache.
+///
+/// Comemo's cache is thread-local, meaning that this only evicts this thread's
+/// cache.
```

Comment:
```suggestion
/// Evict the thread-local cache.
///
/// This only affects functions that are annotated with `#[memoize(local)]`.
```
I also noticed that the copy-pasted comment from the other function is outdated. The thread-local note can be removed from the other one.

I wonder whether the default evict should _also_ evict the thread local cache. The overhead would be quite low if it's unused. I'm also not sure about evicting the accelerator here.

We should think about what the use case of this is vs. evict. If evict does everything, then even if you only use local comemo, evict would do the job. In which use cases does it make sense to have a second evict_local function?